searx/settings.yml

```diff
@@ -1,6 +1,7 @@
 general:
     debug : False # Debug mode, only for development
     instance_name : "searx" # displayed name
+    # contact_url : "mailto:contact@example.com" # contact link on footer
```

Comment:
We can use only one type of *setting*.  If something is added to the *brand*, it is no longer a setting in the `settings.yml`.  Anyway, the rest of this PR looks good to me.

Reply:
Ok, fixed it in c88b067f.